definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2954,5 +2954,17 @@
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/received-service-added-date",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "27/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "makeServiceDecision",
+    "Name": "Make service decision",
+    "Description": "Make service decision",
+    "DisplayOrder": 12,
+    "PreConditionState(s)": "awaitingServiceConsideration",
+    "PostConditionState": "ServiceApplicationNotApproved;AwaitingDNApplication",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
I think you don't need this line as there is no callback. This is only needed for callbacks.